osu.Game.Mode.Osu/Objects/Drawables/Pieces/SliderBody.cs

```diff
@@ -88,6 +81,45 @@ private void load(OsuConfigManager config)
         {
             snakingIn = config.GetBindable<bool>(OsuConfig.SnakingInSliders);
             snakingOut = config.GetBindable<bool>(OsuConfig.SnakingOutSliders);
+
+            int textureWidth = (int)PathWidth * 2;

```

Comment:
yeah, but all the sizes everywhere are weird right now. really need to go through and standardise sizes (one of the things i want to discuss next pair programming session).